source/SkiaSharp.Build.props

```diff
@@ -6,6 +6,7 @@
     <SignAssembly>true</SignAssembly>
     <SkiaSharpBuildPropsImported>true</SkiaSharpBuildPropsImported>
     <MDocDocumentationDirectory>$(MSBuildThisFileDirectory)..\docs\en</MDocDocumentationDirectory>
+    <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
```

Comment:
Since switching to SDK-style projects, this has to be added to to the changes in defaults: traditional projects never generated assembly info, SDK-style projects do.